msal/src/Microsoft.Identity.Client/UIParent.cs

```diff
@@ -50,6 +50,9 @@ public UIParent()
         }
 
 #if iOS
+        /// <summary>
+        /// Constructor for iOS for implementing embedded webview
+        /// </summary>
```

Comment:
not sure if you should add a remark (a real thing as part of the xml doc)/ saying this method is likely to be removed before final release

Reply:
done.